tests/Css/AttributeTranslatorTest.php

```diff
@@ -80,15 +80,16 @@ public static function attributeToStyleTranslationProvider(): array
      *
      * @dataProvider attributeToStyleTranslationProvider
      */
+     #[\PHPUnit\Framework\Attributes\DataProvider('attributeToStyleTranslationProvider')]
     public function testAttributeToStyleTranslation(
         string $body,
         array $expectedStyles
     ): void {
-        $styles = array_fill_keys(array_keys($expectedStyles), []);
+         $styles = array_fill_keys(array_keys($expectedStyles), []);
```

Comment:
This seems to be caused by the extra single-space indent in the attribute lines (5 spaces instead of 4). After removing these extra spaces, phpcs is happy.